bulletin-de-correspondance-hellenique.csl

```diff
@@ -79,7 +79,7 @@
       </group>
     </macro>
     <macro name="page-num">
-      <group delimiter=" ">
+      <group delimiter=" ">
```

Comment:
did you mean to make changes to the spaces here? I can't figure out how they're different -- they both seem like regular spaces, not nbsp

Reply:
It was not my intent, anyway I have inserted back the nbsp if they had disappeared and remove a two spaces sequence
It should be fine now